Index: Modules/ModulesCodingExercise.py
===================================================================
diff --git a/Modules/ModulesCodingExercise.py b/Modules/ModulesCodingExercise.py
deleted file mode 100644
--- a/Modules/ModulesCodingExercise.py	(revision 93d12ca0fc47924ea39c00ac5b69ea87e8894d33)
+++ /dev/null	(revision 93d12ca0fc47924ea39c00ac5b69ea87e8894d33)
@@ -1,8 +0,0 @@
-# Importing the os module
-import os
-
-# Importing math as an alias
-import math as my_math
-
-# Importing everything from random module directly
-from random import *
\ No newline at end of file
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"VcsDirectoryMappings\">\r\n    <mapping directory=\"$PROJECT_DIR$\" vcs=\"Git\" />\r\n    <mapping directory=\"$PROJECT_DIR$/Python_basics\" vcs=\"Git\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
--- a/.idea/vcs.xml	(revision 93d12ca0fc47924ea39c00ac5b69ea87e8894d33)
+++ b/.idea/vcs.xml	(date 1676877667774)
@@ -1,5 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="GithubSharedProjectSettings">
+    <option name="branchProtectionPatterns">
+      <list>
+        <option value="main" />
+      </list>
+    </option>
+  </component>
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
     <mapping directory="$PROJECT_DIR$/Python_basics" vcs="Git" />
diff --git a/Modules/SendingEmailsWithPythonSMTPLIB.py b/Modules/SendingEmailsWithPythonSMTPLIB.py
deleted file mode 100644
